spec/promoted/ruby/client_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.describe Promoted::Ruby::Client do
+  it "has a version number" do
+    expect(Promoted::Ruby::Client::VERSION).not_to be nil
+  end
+
+  it "does something useful" do
```

Comment:
What's this?  Is this a placeholder for tests?

Reply:
yeah this is base layout for test cases.